src/views/viewer/ChannelView/ChannelView.style.tsx

```diff
@@ -64,24 +34,34 @@ export const TitleContainer = styled.div`
 `
 
 export const Title = styled(Text)`
-  ${fluidRange({ prop: 'fontSize', fromSize: '32px', toSize: '40px' })};
+  ${fluidRange({ prop: 'fontSize', fromSize: '24px', toSize: '40px' })};
 
   line-height: 1;
-  padding: ${sizes(1)} ${sizes(2)} ${sizes(2)};
-  background-color: ${colors.gray[800]};
+  margin-bottom: 0;
+  padding: ${sizes(1)} ${sizes(2)} ${0};
```

Comment:
There is no need to use template string here
```suggestion
  padding: ${sizes(1)} ${sizes(2)} 0;
```